.github/workflows/web-analysis.yml

```diff
@@ -38,6 +46,23 @@ on:
       - "!centreon/packages/**"
 
 jobs:
+  dispatch-to-maintained-branches:
+    if: ${{ github.run_attempt == 1 && github.event_name == 'schedule' && github.ref_name == 'develop' }}
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@d632683dd7b4114ad314bca15554477dd762a938 # v4.2.0
+
+      - run: |
+          NIGHTLY_TARGETS=("dev-23.04.x" "dev-23.10.x" "dev-24.04.x" "dev-24.10.x")
```

Comment:
⚠️ This workflow is not merged on dev-24.10.x
It will be, once we merge release-24.10-next

Reply:
As discussed. Missing workflow will be backport ASAP
Thanks

Reply:
This should be addressed here => https://github.com/centreon/centreon/pull/5455 (backport to dev-24.10.x was forgotten)